.github/workflows/step-release-aws.yml

```diff
@@ -21,6 +21,14 @@ on:
         type: number
         required: false
         default: 16
+      sam-template-file:
+        type: string
+        required: false
+        default: ''
+      sam-config-file:
+        type: string
+        required: false
+        default: 'samconfig.toml'
```

Comment:
Doesn't it have to be `./samconfig.toml` ?

Reply:
No, I was a bit unsure about it as well. As it was totally fine with passing the sam-template-file without any path prefix I figured that I could probably remove it from the sam-config-file as well.

[Here](https://github.com/mathem-se/billing-alarms/blob/76e918b1b1f6c1cd326afdd2b138417c25cc98bf/.github/workflows/deploy.yml#L18) you can see me passing a config file without the `./` path prefix